pubspec.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
     sdk: flutter
   pedantic: ^1.8.0
   extra_pedantic: ^1.1.1+3
-  flutter_map: ^0.8.2
+  flutter_map: any
```

Comment:
or use this to prevent any new updates in future 
flutter_map: ">=0.8.0 <1.0.0"